packages/twenty-server/src/engine/core-modules/captcha/captcha.guard.ts

```diff
@@ -26,7 +30,11 @@ export class CaptchaGuard implements CanActivate {
     if (result.success) {
       return true;
     } else {
-      await this.healthCacheService.updateInvalidCaptchaCache(token);
+      await this.metricsService.incrementCounter(
+        MetricsCounterKeys.InvalidCaptcha,
+        [token],
+        MeterKeys.InvalidCaptcha,
```

Comment:
is it needed to have both MetricsCounterKeys and MeterKeys

when passing multiple argument to a function use object parameter pattern:
batchIncrementCounter({ meterKey: MetricsCounterKeys.InvalidCaptcha, eventIds: [] })